### ISSUE TITLE
Activity with a not meeting type is behaving problematic

### ISSUE THREAD

**Issue by TheSlimvReal** [Aam-Digital/ndb-core#1172]
**Describe the bug**
Some of the interaction types configurable enum are marked with `isMeeting: true` which makes the note also show the attendance section (instead of just the participants).
Generally a activity should only be created with a interaction type that is a meeting.
For end-users it is not possible to see which interaction types are a meeting and therefore they might accidentally select a *wrong* interaction type. 
This results in a inconsistent behavior:
1. Attendance can be recorded
2. Attendance is shown in child and activity details
3. Attendance is not shown when recording the same attendance for the same day again
4. A `EventNote` is created which does not show the individuals attendance

**To Reproduce**
1. Go to 'Attendance' and 'Recurring Activity'
2. Click on a 'Coaching Class 2S'
3. Click 'Edit' and select 'Home Visit' as 'Type'
4. Click 'Save'
5. Go to 'Attendance' and 'Record'
6. Click on 'Show more' and select 'Coaching Class 2S'
7. Finish recording the attendance
8. Click on 'Back to Overview' and select the same activity again
9. See now prior attendance is shown

**Expected behavior**
Either it should not be possible to create a activity with a interaction type that is not a meeting or make a `EvenNote` **always** a meeting.

**Comment by sleidig** [comment]
Yes, I have seen this occasionally as well and we should iron out these glitches.

I would like to keep all notes/events behave the same way to avoid confusing people more. i.e. manually created ones and those created as "EventNote" through the "record attendance" feature should have the same display logic.

Only allowing "isMeeting" types to be assigned to RecurringActivity seems a good approach (although you are right that there is no UI indication why some are and some aren't categorized as this).

Alternatively we could have a manual toggle to display linked participants with/without attendance for users (and set its default state based on our "isMeeting" flag and whether there is attendance information already available).


If we find a way to get rid of the "RecurringActivity" layer for users overall, we may have some other screens to make this type more explicit. But I don't have a very clear idea on this yet.

**Comment by TheSlimvReal** [comment]
Some problems have been solved with #1230. Non meeting (`isMeeting === false`) are now still shown in the `RollCallSetupComponent`

**Comment by sleidig** [comment]
now that we have users adding their own categories through the user-editable dropdown options this might become a more common issue. We should probably make the toggle between notes with attendance and notes with simple participant links explicit in the UI. Maybe just a toggle in the Note Details "record attendance for participants" or something?
